app/src/main/java/com/infomaniak/mail/data/cache/mailboxContent/DraftController.kt

```diff
@@ -46,6 +46,7 @@ object DraftController {
     private const val PREFIX_FORWARD = "Fw: "
     private const val REGEX_REPLY = "(re|ref|aw|rif|r):"
     private const val REGEX_FORWARD = "(fw|fwd|rv|wg|tr|i):"
+    private const val REGEX_CID = "<img\\s+src=\"cid:([^\">]+)\">"
```

Comment:
Don't use a regex to parse HTML, directly use Jsoup to parse HTML instead, it's gonna be more robust. This can be expressed using a single css query selector